var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -30,16 +30,21 @@ class Neovim(CMakePackage):
     version('0.2.1', sha256='9e2c068a8994c9023a5f84cde9eb7188d3c85996a7e42e611e3cd0996e345dd3')
     version('0.2.0', sha256='72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5')
```

Comment:
minor: what about adding a comment to not use previous versions? do you think it is reasonable?

https://github.com/neovim/neovim/wiki/FAQ#can-i-use-lua-based-vim-plugins-eg-neocomplete

Reply:
Yes, in fact what do you think of explicitly deprecating older versions with the keyword argument? They would still be buildable but at least people would get a warning.

Reply:
I'm in!